src/web/components/Core/Form.tsx

```diff
@@ -15,6 +22,11 @@ type FormProps<T extends FieldValues> = {
   id?: string;
 };
 
+export function withoutRef<TFieldName extends string>(props: UseFormRegisterReturn<TFieldName>) {
```

Comment:
Helper for `register` - it always provides `ref` but not all form-related components need it, and passing it unnecessarily requires forwardRef (which breaks TS types).